src/webgpu/shader/validation/wgsl/switch-must-have-exactly-one-default-clause.fail.wgsl

```diff
@@ -0,0 +1,11 @@
+# v-0008: line 6: switch statement must have exactly one default clause
```

Comment:
I keep losing track of this one - is this what we agreed? I thought it was okay to not have a default clause (the semantics should be pretty obvious I think). Not trying to change this, just want to make sure it's the right thing.

Reply:
I believe it's `at most one` although the spec isn't updated yet. @dneto0 to verify my memory.

Reply:
I'm wrong, it's must have currently.